services/user.py

```diff
@@ -0,0 +1,41 @@
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None,
+) -> User:
+
+    user = User.objects.create_user(
+        username=username,
+        password=password,
+    )
+
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
```

Comment:
The `get_user` function uses `User.objects.get()`, which will raise a `DoesNotExist` exception if the user is not found. Consider handling this exception to prevent potential crashes.